pynest/examples/Potjans_2014/network.py

```diff
@@ -199,26 +211,32 @@ def create_populations(self):
                     'I_e': self.DC_amp_e[i]
                     }
                 )
+            if self.net_dict['V0_type'] == 'optimized':
+                nest.SetStatus(population, 'V_m', np.random.normal(
```

Comment:
This breaks the correct seeding of random number generators in a distributed simulation, see https://nest-simulator.org/random_numbers/
I'd suggest to copy the loop over the threads (line 223) here and replace `['V0_mean']['original']` by `['V0_mean']['optimized'][i]` and the same for `V0_sd`.